cluster/addons/volumesnapshots/rbac/rbac-common.yaml

```diff
@@ -0,0 +1,100 @@
+# RBAC file for the common external snapshot controller.
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: common-csi-snapshotter
```

Comment:
does it make sense to just call them "common-snapshot-controller" and "sidecar-snapshotter"?

Reply:
I would just call it "snapshot-controller"

Reply:
This is really to help me understanding the structure of the whole storage test suite better.
It looks to me rbac files are managed under testing-manifests/storage-csi. I also found deployment yamls under storage-csi/hostpath/hostpath (not for others though). It's a bit hard for me to understand how yamls are organized under testing-manifests/storage-csi, is it the place holder for all yaml files? if thats the case, does it make more sense to move all yaml files to there? @msau42 maybe you could shed some light on this?

Reply:
cluster/addons is the place to put production manifests.  So something like the snapshot controller should go here because the deployer is responsible for managing it.

Things that are only needed for e2e testing go under testing-manifests/csi.  Hostpath csi driver lives there because the driver is only meant for testing purposes and not for use in a real production cluster.

Production csi drivers like PD *should* theoretically go under cluster/addons too, but considering the whole cloud-provider-out-of-tree effort, an out-of-tree addon manager may be a better home for it.

Reply:
@msau42, "So something like the snapshot controller should go here because the deployer is responsible for managing it.", my understanding is that even if you have the controller manifests under cluster/addons, its still deployer's decision whether or not to include it during a cluster creation. For cluster/gce, ENABLE_SNAPSHOTS_CRDS controls whether it should be installed. Is this accurate?

Reply:
correct